tools/ScenarioTest.ResourceManager/ScenarioTest.ResourceManager.csproj

```diff
@@ -0,0 +1,39 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  
+  <Import Project="$(MSBuildThisFileDirectory)..\..\src\Az.Test.props" />
+
+  <PropertyGroup>
+    <AssemblyName>Microsoft.Azure.PowerShell.ScenarioTest.ResourceManager</AssemblyName>
+    <RootNamespace>$(LegacyAssemblyPrefix)ScenarioTest</RootNamespace>
+    <IncludeCommonTestProject>false</IncludeCommonTestProject>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Moq" Version="4.7.145" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <ProjectReference Include="..\..\src\Accounts\Authentication\Authentication.csproj" />
```

Comment:
Please use `Include="$(RepoSrc)Accounts\Authentication\Authentication.csproj". Same for the other project reference.